### DISCUSSION TITLE
Why .NET CORE 5 works with IIS EXPRESS on ARM64 and .NET CORE 6 dont?

### DISCUSSION THREAD

**Discussion by kn04nd** [dotnet/aspnetcore#56414] [Q&A]
Hi Folks!

I've working with .net core with many versions, and actually I'm using a MacBook M1 Pro

I use Parallels for all my developments and works fine!

But, recently I encountered a curious thing...

If I create a basic api project with .NET CORE 5 using Visual Studio 2019 and try to run using IIS Express 10, runs perfectly!

Buut,

if I create another project, same basic api project with .NET CORE 6, now using Visual Studio 2022 (2019 can't runs .net core 6), and try to run using IIS Express 10, a get an error like this: "IIS EXPRESS on arm64 is only supported for ASP NET Core applications targeting .NET 7.0 and newer"

Why this?
Someone can help me?

**Comment by danmoseley** [comment]
Can you use .NET 8?

**Reply by kn04nd** [reply]
No, I can't, I have an old solution using .net core 6.

That is the reason why I created this discussion :/

**Reply by danmoseley** [reply]
I see. Note it will be out of support soon. Can you test at least on .NET 8?

https://devblogs.microsoft.com/dotnet/dotnet-6-end-of-support/
